entrypoint.sh

```diff
@@ -5,7 +5,9 @@ LOG_LEVEL="${LOG_LEVEL:-info}"
 SET_PERMISSIONS="${SET_PERMISSIONS:-false}"
 SHARE_NAME="${SHARE_NAME:-TimeMachine}"
 CUSTOM_AFP_CONF="${CUSTOM_AFP_CONF:-false}"
+CUSTOM_SMB_AUTH="${CUSTOM_SMB_AUTH:-no}"
```

Comment:
Based on the current config from my time machine container, it looks like this should default to `ntlmv2-only`:

```
# testparm -v | grep "ntlm auth"
Load smb config files from /etc/samba/smb.conf
Loaded services file OK.
Weak crypto is allowed

Server role: ROLE_STANDALONE

Press enter to see a dump of your service definitions

        ntlm auth = ntlmv2-only
```

Reply:
`yes` and `no` are aliases, accordingly to Samba dos:
https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#idm7319

Reply:
Ah, gotcha.  Thanks!  I should have RTFM.